src/mtev_cluster.c

```diff
@@ -65,13 +82,16 @@ struct mtev_cluster_t {
   mtev_cht_t **cht;
   mtev_net_heartbeat_ctx *hbctx;
   mtev_cluster_node_t *oldest_node;
+  struct jl_array_list *hb_payloads;
```

Comment:
json-lib is third-party code that we've name-spaced... but we try to keep the changes to an absolutely minimum so that we can keep it up to date with the 3rdparty upstream.  Renaming this isn't an option.  Reimplementing this to be part of utils is an option, but I personally am not a fan of convenience functions for the most simple of data structure manipulations.